test/spmd/test_xla_virtual_device.py

```diff
@@ -42,16 +42,6 @@ def test_metrics_recorded(self):
     self.assertIn("VirtualDeviceUsage", met.counter_names())
     self.assertNotEqual(met.counter_value("VirtualDeviceUsage"), 0)
 
-  @unittest.skip("fails with functionalization")
```

Comment:
Rather than removing the test, should we add a metric for the other codepath (that handles IR) and check that it's used?